src/promptflow/promptflow/_core/tool_meta_generator.py

```diff
@@ -151,13 +162,22 @@ def load_python_module_from_file(src_file: Path):
     src_file = Path(src_file).resolve()  # Make sure the path is absolute to align with python import behavior.
     spec = importlib.util.spec_from_file_location("__pf_main__", location=src_file)
     if spec is None or spec.loader is None:
-        raise PythonLoaderNotFound(f"Failed to load python file '{src_file}', please make sure it is a valid .py file.")
+        raise PythonLoaderNotFound(
+            message_format="Failed to load python file '{src_file}'. Please make sure it is a valid .py file.",
+            src_file=src_file,
+        )
     m = importlib.util.module_from_spec(spec)
     try:
         spec.loader.exec_module(m)
     except Exception as e:
         # TODO: add stacktrace to additional info
-        raise PythonLoadError(f"Failed to load python module from file '{src_file}', reason: {e}.") from e
+        error_type_and_message = f"({e.__class__.__name__}) {e}"
+        raise PythonLoadError(
+            message_format="Failed to load python module from file '{src_file}'. "
+            "Error details: {error_type_and_message}",
```

Comment:
Let's use this style:
`Failed to load python module from file '{src_file}': {error_type_and_message}`

Reply:
Done.